src/components/BookCategory.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkBookStatus } from 'redux/categories/categoriesSlice';
+
+function BookCategory() {
+  const dispatch = useDispatch();
+  function handleClick() {
+    dispatch(checkBookStatus);
+  }
+  return (
+    <div className="wrapper">
+      <button type="button" onClick={handleClick}>
+        Check Status
+      </button>
+    </div>
+  );
```

Comment:
- [ ] Kindly make sure to display the message `UNDER CONSTRUCTION` when the `Check Status` button is clicked. At this moment there is nothing displayed when the button is clicked. Kindly fix that.